superset/config.py

```diff
@@ -991,3 +991,8 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
     except Exception:
         logger.exception("Found but failed to import local superset_config")
         raise
+
+
+# It's possible to add a dataset health check logic which is specific to your system.
+# It will get executed each time when user open a chart's explore view.
+DATASET_HEALTH_CHECK = None
```

Comment:
can you add the signature about the health check to the comment here?

Or maybe even better, include a default function that always passes. It's actually a bit unclear to me how this should be used (should I return True if the dataset passes? None?)

Reply:
See Jesse's comment above, if the dataset passes,  it will still add a record into db record (so that not running check again).